yarn.lock

```diff
@@ -1711,9 +1716,9 @@
     "@types/responselike" "*"
 
 "@types/fs-extra@^9.0.1":
-  version "9.0.1"
-  resolved "https://registry.npmjs.org/@types/fs-extra/-/fs-extra-9.0.1.tgz#91c8fc4c51f6d5dbe44c2ca9ab09310bd00c7918"
-  integrity sha512-B42Sxuaz09MhC3DDeW5kubRcQ5by4iuVQ0cRRWM2lggLzAa/KVom0Aft/208NgMvNQQZ86s5rVcqDdn/SH0/mg==
+  version "9.0.2"
```

Comment:
Just for my understanding - did you delete the yarn.lock? I thought changes in the lock file should only be new additions, but not the old versions, but maybe I am mistaken 🤔

Reply:
The pipeline failed due to `check:test-service`, which detected the changes in the `yarn.lock`.
I tried to install again, which did not help:
I also tried to run the script locally, no errors were shown.
Therefore, I deleted the lock file completely and re-generated it again.